src/riscv-integration.adoc

```diff
@@ -1058,7 +1059,7 @@ NOTE: `auth_cap` is <<ddc>> for Legacy mode and `cs1` for Capability Mode
 6+| *Store/atomic/cache-block-operation additional exception checks*
 | all stores, all atomics, all cbos | {cheri_excep_mcause} | {cheri_excep_type_data} | {cheri_excep_cause_tag}    |`auth_cap` tag        | not(`auth_cap.tag`)
 | all stores, all atomics, all cbos | {cheri_excep_mcause} | {cheri_excep_type_data} | {cheri_excep_cause_seal}   |`auth_cap` seal       | isCapSealed(`auth_cap`)
-|             all atomics, all cbos | {cheri_excep_mcause} | {cheri_excep_type_data} | {cheri_excep_cause_perm}   |`auth_cap` permission | AMO only: not(`auth_cap`.<<r_perm>>)
+|             all atomics           | {cheri_excep_mcause} | {cheri_excep_type_data} | {cheri_excep_cause_perm}   |`auth_cap` permission | not(`auth_cap`.<<r_perm>>)
```

Comment:
"Non-prefetch CBOs"?

Reply:
CBO.ZERO is equivalent to a series of writes, so it shouldn't check R.  FLUSH/INVAL are less critical but are more writey than ready.

Reply:
There were previous discussions about what permissions to check for each CBO. The checks are summarised [here](https://github.com/CTSRD-CHERI/cheri-specification/issues/65#issuecomment-1721012520) for all CBO and prefetch instructions:
* cbo.zero is a whole cache line store, so it behaves and is checked exactly like any other RISC-V store
    * Bounds, tag, store permission, etc are checked
    * There are two variants of the instruction depending on the PCC mode flag
* prefetch.i, prefetch.r and prefetch.w
    * Tag and sealing check as usual
    * prefetch.i check execute permission
    * prefetch.w check store permission
    * prefetch.r check load permission
    * Check that the authorising capability's bounds cover at least 1 byte in the cache line prefetched
    * Do not start the prefetch if any of the checks above fails.
* cbo.clean and cbo.flush
    * Tag and sealing check as usual
    * Check that there are store or load permissions
    * Check that the authorising capability's bounds cover at least 1 byte in the cache line flushed/cleaned
* cbo.inval
    * Tag and sealing check as usual
    * Check that there are store, load and ASR permissions
    * Check that the authorising capability's bounds cover the _entire_ cache line invalidated
    * The same checks must be retained if cbo.inval is configured to behave as cbo.flush